src/components/PanelHeader/PanelHeader.tsx

```diff
@@ -1,19 +1,32 @@
 import { ReactNode } from "react";
-import { useQueryParam, StringParam } from "use-query-params";
+import { useQueryParams, StringParam } from "use-query-params";
 
 type Props = {
   title: ReactNode;
 };
 
+type QueryString = {
+  [key: string]: string | undefined;
+};
+
 export default function PanelHeader({ title }: Props): JSX.Element {
-  const [panelQs, setPanelQs] = useQueryParam("panel", StringParam);
+  const [panelQs, setPanelQs] = useQueryParams({
+    panel: StringParam,
+    model: StringParam,
+  });
+
+  // Remove all query strings when close button is clicked
```

Comment:
I'm not 100% sure but I think `panelQs` will only have `panel` and `model` regardless of the rest of the query params. If so then this is working correctly and this comment just needs to be updated. If this isn't the case then you'll only want to clear those 2 params and not any other in there.

Can you verify?

Reply:
It will only have panel and model but I purposely did it like this on the assumption that other panels may need other query params so clearing the panel would clear them all? If you still think this should only clear the two query params, ping me and I'll change.